data/wanderer dire storm.txt

```diff
@@ -0,0 +1,136 @@
+
+ship "Dire Storm"
+	sprite "ship/dire storm"
+	attributes
+		category "Heavy Warship"
+		licenses
+			"Wanderer Military"
+		"cost" 40500000
+		"shields" 54128
+		"hull" 48962
+		"required crew" 72
+		"bunks" 237
+		"mass" 832
+		"drag" 13.9
+		"heat dissipation" 0.5
+		"fuel capacity" 816
+		"cargo space" 424
+		"outfit space" 864
+		"weapon capacity" 276
+		"engine capacity" 240
+		weapon
+			"blast radius" 340
+			"shield damage" 3400
+			"hull damage" 1700
+			"hit force" 5100
+	outfits
+		"Sunbeam" 2
+		"Thunderhead Launcher" 2
+		"Thunderhead Missile" 80
+		"Sunbeam Turret" 2
+		"Wanderer Anti-Missile" 2
+		
+		"Blue Sun Reactor"
+		"Large Biochemical Cell"
+		"Dark Storm Shielding" 2
+		"Bright Cloud Shielding"
+		"Wanderer Ramscoop" 2
+		"Fuel Pod" 2
+		
+		"Type 4 Radiant Thruster" 2
+		"Type 4 Radiant Steering" 2
+		"Hyperdrive"
+		
+	engine -98 160
+	engine 98 160
+	engine 0 289
+	engine -62 232
+	engine 62 232
+	gun -12 -156 "Sunbeam"
+	gun 12 -156 "Sunbeam"
+	gun -17 -134 "Thunderhead Launcher"
+	gun 17 -134 "Thunderhead Launcher"
+	turret -34 16 "Sunbeam Turret"
+	turret 34 16 "Sunbeam Turret"
+	turret -38 105 "Wanderer Anti-Missile"
+	turret 38 105 "Wanderer Anti-Missile"
+	drone 84 63
+	drone -84 63
+	drone -72 63
+	drone 72 63
+	explode "small explosion" 40
+	explode "medium explosion" 70
+	explode "large explosion" 50
+	explode "huge explosion" 20
+	"final explode" "final explosion large" 1
+	description "The Dire Storm is a deep space command, control and science vessel, designed to be one of the first ships through the Eye, to begin cataloging the dangers of the unknown space beyond, and to provide a secure, mobile headquarters for the warships that stay behind to cover the Wanderers' retreat. Endless banks of climate-controlled storage allow the dozen (or so) science labs to take hundreds of soil, water and even biological specimens from desolate worlds. Four drones allow remote, efficient harvesting and in an emergency can provide cover for a fleet of ships."
+	
+ship "Dire Storm" "Dire Storm (Passenger)"
+	outfits
+		"Thunderhead Launcher" 4
+		"Thunderhead Missile" 160
+		"Dual Sunbeam Turret"
+		"Wanderer Anti-Missile" 2
+		
+		"White Sun Reactor"
+		"Dark Storm Shielding" 2
+		"Wanderer Ramscoop"
+		"Outfits Expansion" 14
+		"Bunk Room" 23
+		
+		"Type 4 Radiant Thruster"
+		"Type 1 Radiant Thruster"
+		"Type 4 Radiant Steering"
+		"Type 1 Radiant Steering"
+		"Hyperdrive"
+
+	gun "Thunderhead Launcher"
+	gun "Thunderhead Launcher"
+	gun "Thunderhead Launcher"
+	gun "Thunderhead Launcher"
+	turret "Dual Sunbeam Turret"
+	turret
+	turret "Wanderer Anti-Missile"
+	turret "Wanderer Anti-Missile"
+	
+event "wanderers: hurricane mass production"
+	fleet "Wanderer Defense"
```

Comment:
If this is a science ship with dozens of climate controlled labs, why is it a part of planetary defense fleets?

Reply:
It's a command and control vessel. Think Enterprise.

Reply:
I'm as big a Star Trek fan as they come, but the Enterprise doesn't really make that much logical sense even in its own universe. It's a flagship that usually operates alone. It's the heaviest warship in the Federation fleet that's often called upon to go into battle, yet it houses a significant civilian population as well as a large amount of staff and equipment for scientific endeavors, all of whom are put in great and unnecessary risk every time the ship goes into battle.

Realistically, this kind of jack-of-all-trades vessel makes no sense. It makes far more sense to have more than one ship, each optimized for its own role. For example, to do science, you would want a dedicated science ship. To _briefly_ do science in a hostile location, you would want your science ship protected with a fleet of dedicated military warships so it can _quickly_ do the science and then get out of there ASAP. Or you would want the science ship to be incredibly light and fast. Like, say, a Scout or a Bounder.

The only reason why the jack-of-all-trades Bactrian exists in Endless Sky is because it hearkens back from a brief time when a self-sufficient warship/freighter/transport _did_ make sense. But it's a relic--a product of an older time marked by exploration and colonization in regions of space with no military support. That doesn't describe the Wanderers' situation.

Reply:
The Wanderers don't make plans to briefly do anything. They go into hostile territory, and they settle there for thousands of years. They have no military experience and no interest in getting military experience, but they do plan to survive in harsh conditions that literally killed the previous inhabitants of the sector. 

If you don't think the wanderers are explorers, scientists and settlers that's totally fine, but let's agree to differ on this point. You don't think the ship is needed. Thank you for your opinion.

Reply:
The wanderers certainly are explorers, scientists and settlers. My contention is that this multipurpose vessel does not make logical sense.

You're misunderstanding why self-sufficient jack-of-all-trade ships exist; the only logical reason is because there are no armed escorts available. Any time escorts are available, it makes more sense to have each ship perform a dedicated specialist role rather than try to generalize. To settle in potentially hostile lands, you'd have an unarmed yet well defended colony ship escorted by a military task force to protect it. The only reason to put a ton of guns and engines on the colony ship is if there was no military escort available--which was the case for the Bactrian. But it is not the case for the Wanderers as they pass through the Eye. They have plenty of warships available to protect their vulnerable transports and cargo ships.